bindata/network/multus/multus.yaml

```diff
@@ -783,4 +805,74 @@ spec:
             items:
             - key: reconciler_cron_expression
               path: config
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: whereabouts-controller
+  namespace: openshift-multus
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: whereabouts-controller
+  template:
+    metadata:
+      labels:
+        app: whereabouts-controller
+    spec:
+      containers:
+        - command:
+            - /usr/src/whereabouts/bin/node-slice-controller
+          env:
+            - name: NODENAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+            - name: WHEREABOLaunch nightly aws,no-spotUTS_NAMESPACE
```

Comment:
OOPS! Dude, this has a funky copy/pasta error

Reply:
good catch 
will take care of it